docs/langgraph/README.md

```diff
@@ -0,0 +1,46 @@
+# LangGraph.js Complete Guide
+
+This directory contains a comprehensive guide for LangGraph.js - a low-level orchestration framework for building controllable agents.
+
+## 📁 Documentation Structure
+
+### Getting Started
+- **[installation.md](installation.md)** - Installation and setup
+
+### Core Concepts
+- **[core-concepts.md](core-concepts.md)** - Graph state definition, basic construction, and node patterns
+- **[control-flow.md](control-flow.md)** - Branching, map-reduce, conditional routing, and loop control
+- **[state-management.md](state-management.md)** - State schema design, updates, and persistence patterns
+
+### Advanced Features
+- **[persistence-memory.md](persistence-memory.md)** - Thread-level persistence, memory management, and subgraph persistence
+- **[human-in-the-loop.md](human-in-the-loop.md)** - Interrupts, tool call review, breakpoints, and time travel
```

Comment:
Since this has not yet been implemented, I believe it is better not to include it.

Reply:
You were right! I've removed the documentation for the undeveloped features.
https://github.com/liam-hq/liam/pull/3486/commits/e310ff48ec18b8fb3f43119a5b418ecb631ca538